scripts/collect-search-results.js

```diff
@@ -34,7 +34,11 @@ async function main({args}) {
         BUCKET: `nextstrain-data`,
         fileToUrl: (filename) => `https://nextstrain.org/${filename.replace('.json', '').replace(/_/g, '/')}`,
         inclusionTest: (fn) => {
-          return (fn.startsWith("ncov_") && !fn.endsWith("_gisaid.json") && !fn.endsWith("_zh.json"));
+          return (fn.startsWith("ncov_")
+            && !fn.endsWith("_gisaid.json")
+            && !fn.endsWith("_zh.json")
+            // Exclude all dated JSONs to avoid OOM error
+            && !fn.match(new RegExp(/_[0-9]{4}-[0-9]{2}-[0-9]{2}\.json/)));
```

Comment:
Would be good to anchor this pattern to the end of the name, e.g.

```suggestion
            && !fn.match(new RegExp(/_[0-9]{4}-[0-9]{2}-[0-9]{2}\.json$/)));
```

also, FYI, you can use regex literal syntax:

```suggestion
            && !fn.match(/_[0-9]{4}-[0-9]{2}-[0-9]{2}\.json$/));
```

Reply:
Ah, thanks! Updated in force push.